packages/my-joy-beta/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Joyent",
+  "name": "My Joyent &beta;",
+  "icons": [
+    {
+      "src": "favicon.ico",
```

Comment:
Yoou should have more icons , this will help all phones add it to homescreen:
https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/

Reply:
yes, I know. I haven't asked for those icons yet :/

Reply:
😱